src/components/ProfileTree/ProfileLinker/LinkerMenu.js

```diff
@@ -53,6 +60,7 @@ LinkerMenu.propTypes = {
   onToggle: PropTypes.func.isRequired,
   onClick: PropTypes.func.isRequired,
   keyHandler: PropTypes.func,
+  disabledOptions: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Shouldn't this value be required

Reply:
I guess this prop is not required, added this to defaultProps.